docs/customizing_dashboards.md

```diff
@@ -74,6 +74,10 @@ which are specified through the `.with_options` class method:
 `:limit` - Set the number of resources to display in the show view. Default is
 `5`.
 
+`:sort_by` - What to sort the association by in the show view.
+
+`:direction` - What direction the sort should be in, `:asc` (default) or `:desc`.
```

Comment:
Do you mean `direction` -> `order`? I chose direction in order to keep things consistent with the naming in `Administrate::Order`.